### PR TITLE
Prepare amq-streams12 files

### PR DESCRIPTION
**Pull request by stanlyDoge**

### CODE REVIEW

**Comment by kyguy** [comment]
+1 @ppatierno

File: operator/image.yaml [hunk 1]
```diff
@@ -31,6 +33,8 @@ modules:
         ref: sprint-28
     - name: modules
       path: modules
+    - name: manifests
+      path: manifests
```
Comment (kyguy) [inline operator/image.yaml]: After creating an "olm" module ( refer to @ppatierno 's comment ), we can install it using
```
-name: olm
```
the same way we install the "operator" module [1]. We won't need these two lines


[1] https://github.com/jboss-container-images/amqstreams-1-openshift-image/blob/f785f9801f25ac7dc95c7e4e8d884c4e9ead4ae6/operator/image.yaml#L43

File: operator/modules/olm/install.sh [hunk 2]
```diff
@@ -5,9 +5,9 @@ SCRIPT_DIR=$(dirname $0)
...
-mkdir -p ${STRIMZI_HOME}/manifests
+mkdir -p /manifests
 
 # copy manifests files
-cp -r ${MANIFESTS_DIR}/* ${STRIMZI_HOME}/manifests
+cp -r ${MANIFESTS_DIR}/* /manifests
 
-chmod -R 0755 ${STRIMZI_HOME}/manifests
\ No newline at end of file
+chmod -R 0755 /manifests
```
Comment (ppatierno) [inline operator/modules/olm/install.sh]: I don't think they are executable files, or?

**Comment by ppatierno** [comment]
@stanlyDoge I did the verification and they are valid.

**Pull request merged**